src/Tokenizers/MySQL/ColumnTokenizer.php

```diff
@@ -133,13 +136,28 @@ protected function consumeDefaultValue()
                     }
                 } else {
                     $this->definition->setDefaultValue((string) $this->definition->getDefaultValue());
+                    $this->definition->setNullable(false);
```

Comment:
Hmm. This seems to be causing some issues with the tests. I'm not sure this makes sense to always mark something that has a default value as nullable. Nullable should get set by the table having "NOT NULL" in its column schema.

Reply:
I wasn't sure how to do this.  If the column sql/definition is

```
$table->char('country', 2)->default('US');
```

This will be "not nullable" but this package will generate

```
$table->char('country', 2)->nullable()->default('US');
```

without line #139

Reply:
Interesting. I think technically this example could still have a null value, since it's not marked as not nullable (`->nullable(false)`). I would need to look further into this to see what implicit null means in MySQL for each data type, but I think all columns CAN be null unless its definition includes "NOT NULL", which in your case it does not.

Reply:
I suppose if everything is by default nullable and wasn't something that came in explicitly from the column's schema, it could exclude that `nullable()` so that it's like the original schema where it's just implied that it's nullable.

Reply:
Do you want to make the change?  Or I can just remove line 139 from my PR.

Reply:
Yeah, I'll probably end up making some changes to the column definition to allow for the implicit nulls.